src/Plutarch/Extra/AssetClass.hs

```diff
@@ -150,9 +158,9 @@ passetClass = phoistAcyclic $
   plam $ \sym tk ->
     pcon $ PAssetClass (pdata sym) (pdata tk)
 
--- | @since 3.10.0
+-- | @since 3.10.3
 passetClassT ::
-  forall (unit :: Symbol) (s :: S).
+  forall {k} (unit :: k) (s :: S).
```

Comment:
I'd rather an explicit than implicit argument here.

Reply:
We moved away from `Symbol` to use type tags as well. How would an explicit polykind look like?

Reply:
`forall (k :: Type) (unit :: k) ...`

`Type :: Type` and all that.

Reply:
It might be less convenient in that ordering, so maybe `forall (unit :: k) (k :: Type) ...` would be better.

Reply:
:bulb: Ah, that makes sense.

Reply:
`forall (unit :: k) (k :: Type)` won't work cause GHC can't find `k`.

Reply:
I guess we can make `k` inference only?

Reply:
Inference only for `k` makes sense to me here. We definitely want to specify the unit first.

Reply:
curious what the practical difference is between `Symbol` and just regular type tags. Obviously the latter is more polymorphic, but when would we want to _not_ use a `Symbol` in practice?

Reply:
See https://github.com/Liqwid-Labs/agora/blob/205241a7ad6b7f631aa670969cc25ef3128d09b8/agora/Agora/SafeMoney.hs